### PR TITLE
[docs]: Fix documentation for Button Group Component

### PR DESCRIPTION
**Pull request by HareeshDhruva**
Here's a sample Pull Request (PR) description for the changes made to the Button Group documentation:
PR Title

Update Button Group Documentation: Properties, Events, Exposed Variables, and Styles Enhancements
Description

This PR improves the Button Group component documentation by enhancing clarity and consistency across Properties, Events, Exposed Variables, Devices, and Styles sections. Updates include detailed explanations, expected values, and new examples to make it easier for users to understand and configure the component.
Changes Made

fixes issue : #11181

### CODE REVIEW

**Comment by PriteshKiri** [comment]
Hey @HareeshDhruva,

Thanks for your contribution.

Can you please make the changes in both the files mentioned in the issue description?

Also, you have to update the content of the tables with respect to the given references only. You can replicate the sentence structure in the given references.

**Comment by HareeshDhruva** [comment]
I noticed the code structure differs between these files. Are we only focusing on updating the content, or should I also align the structure to keep both files consistent?

**Comment by PriteshKiri** [comment]
@HareeshDhruva we just need to update the content. The content style/structure should be consistent.

**Comment by HareeshDhruva** [comment]
"I've submitted a new PR for this issue that includes additional updates. Please review it when you have a chance!" 
This PR is for issue #11184

File: docs/docs/widgets/button-group.md [hunk 1]
```diff
@@ -18,20 +18,18 @@ The **Button Group** component is used to group a series of buttons together in
...
 
 <div style={{paddingTop:'24px', paddingBottom:'24px'}}>
 
 ## Events
 
-Events are actions that can be triggered programmatically when the user interacts with the component. Click on the component handle to open its properties on the right. Go to the **Events** accordion and click on **+ Add handler**.
-
 | <div style={{ width:"100px"}}> Events </div> | <div style={{ width:"100px"}}> Description </div> |
 |:----------- |:----------- |
 | On click | This event is triggered when the user clicks on the button in the Button Group. |
```
Comment (RudraPratik30) [inline docs/docs/widgets/button-group.md:35]: Hey @HareeshDhruva, as mentioned in the issue, you have to update the content of the description column here. ([reference](https://docs.tooljet.com/docs/widgets/text-input/#events) (Please just check the sentence structure from the given reference))

File: docs/docs/widgets/button-group.md [hunk 2]
```diff
@@ -56,7 +54,7 @@ There are currently no CSA (Component-Specific Actions) implemented to regulate
 
 | <div style={{ width:"100px"}}> Variable </div> | <div style={{ width:"200px"}}> Description </div>| <div style={{ width:"135px"}}> How To Access </div> |
 | :---------- | :---------- | :----------|
-| selected | If the **enable multiple selection** option is turned off, then the variable is an array of objects, and the first object holds the value of the selected button. However, if it is turned on, the variable type changes from an array to an object, and the selected button values are stored as a string within that object. | Access the value using `{{components.buttongroup1.selected[0]}}` or `{{components.buttongroup1.selected}}` |
+| selected | If the **enable multiple selection** option is turned off, then the variable is an array of objects, and the first object holds the value of the selected button. However, if it is turned on, the variable type changes from an array to an object, and the selected button values are stored as a string within that object. | Access the value using `{{components.buttongroup1.selected[0]}}` or `{{components.buttongroup1.selected}}`. |
```
Comment (RudraPratik30) [inline docs/docs/widgets/button-group.md:57]: Here as well, you need to update the content of description and how to access column. ([reference](https://docs.tooljet.com/docs/widgets/text-input/#exposed-variables) ( Please just check the sentence structure from the given reference ))

File: docs/docs/widgets/button-group.md [hunk 3]
```diff
@@ -77,12 +75,12 @@ A Tooltip is often used to display additional information when the user hovers t
...
-## Layout
+## Devices
 
-| <div style={{ width:"100px"}}> Layout </div> | <div style={{ width:"100px"}}> Description </div> | 
-| :----------- | :----------- | 
-| Desktop | Toggle to show or hide the component in the desktop view. Dynamically configure the value by clicking on `Fx` and entering a logical expression that results in either true or false. Alternatively, the  values can be set to **`{{true}}`** or **`{{false}}`**.|  
-| Mobile | Toggle to show or hide the component in the desktop view. Dynamically configure the value by clicking on `Fx` and entering a logical expression that results in either true or false. Alternatively, the  values can be set to **`{{true}}`** or **`{{false}}`**. | 
+| <div style={{ width:"100px"}}> Property </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"135px"}}> Expected Value </div> |
+| :----------- | :----------- | :----------- |
+| Desktop | Toggle to show or hide the component in the desktop view. Dynamically configure the value by clicking on `Fx` and entering a logical expression that results in either `true` or `false`. | **`{{true}}`** or **`{{false}}`**. |
```
Comment (RudraPratik30) [inline docs/docs/widgets/button-group.md:82]: Update the content in the "Description" and "Expected value" columns as per the given [reference](https://docs.tooljet.com/docs/widgets/text-input/#devices) ( Please just check the sentence structure from the given reference )

File: docs/docs/widgets/button-group.md (hunk 1 above)
Comment (HareeshDhruva) [reply docs/docs/widgets/button-group.md:35]: I noticed that clicking the "reference" redirects to the text input components section. To ensure consistency, I would like to follow the sentence structure used there for the event descriptions in our documentation.

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]
Hey @HareeshDhruva, I just noticed that you have made changes in the wrong file, as per the issue you were supposed to make changes in the Next and LTS version, but I can see instead of 2.50.0 (LTS) version you have edited 2.5.0 version by mistake.

Can you please revert all the changes from 2.5.0 and make the correct changes in 2.50.0 LTS version.

**Comment by HareeshDhruva** [comment]
I reverted the file to its original forked version and applied recent changes to another file.

**Review by RudraPratik30 — DISMISSED** [review]
Hey @HareeshDhruva, thanks for contributing!
The changes look good to me, forwarding it to the team for final review.
Your PR will be merged soon, thanks!

**Comment by PriteshKiri** [comment]
Hey @HareeshDhruva,

I see that some correct changes were reverted. Could you please address this so we can move forward and merge the PR after approval?

**Comment by HareeshDhruva** [comment]
I made another PR for the same issue

**Comment by HareeshDhruva** [comment]
> Hey @HareeshDhruva,
> 
> I see that some correct changes were reverted. Could you please address this so we can move forward and merge the PR after approval?

Is it okay ?

**Comment by RudraPratik30** [comment]
Hey @HareeshDhruva, I can see that you have opened an [another PR](https://github.com/ToolJet/ToolJet/pull/11216) for the same issue, which is already approved by me. So you can close this PR if it is no longer needed.

**Pull request closed without merging**